sw/device/silicon_creator/rom_exts/meson.build

```diff
@@ -70,23 +70,19 @@ foreach slot, slot_link_args : rom_ext_link_info
 endforeach
 
 # ROM_EXT manifest generator.
-rom_exts_manifest_offsets_header = custom_target(
-  'rom_exts_manifest_h',
-  output: 'manifest.h',
-  depend_files: [
-    'manifest.h.tpl',
-    'manifest.hjson',
-    meson.source_root() / 'util/rom-ext-manifest-generator.py',
-  ],
-  command: [
-    prog_python,
+rom_exts_manifest_hdr_gen = generator(
```

Comment:
Interesting.  Is there a simple explanation why the generator successfully forces this but the custom_target does not?   (I took a hand at this my self, but did not switch to a generator, and then had little success) Regardless, this now works much better.  Thank you!

Reply:
The documentation is less than clear about generators vs. custom_targets for emitting things like header files (and in fact, recommends a generator, and then a custom target in the very next paragraph).

My intuition is that the problem was that `rom_exts_manifest_offsets_header` was missing from the sources above, so including the file would succeed or fail based on the whether or not the header happened to be created before the `mask_rom_lib` target would have run.

Reply:
Thanks @cfrantz.  IIRC the issue I had is that when I simply used placed the name of the custom_target object in the adjacent file it was not recognized.  I presumed that had something do with scoping of objects in meson.  Regardless, I'll be glad to have this merged.

Reply:
BTW, here is the documentation I was referring to:
https://mesonbuild.com/Generating-sources.html#using-generator

You can see the second and third paragraphs seem to offer conflicting (or at least confusing) advice.  I _think_ the generator is maybe a more parameterizable target, as the generator rule contains `@`-substitutions and the target rule invokes `gen()` with parameters.

Reply:
SG. I reverted this change and went with @cfrantz recommendation.